exercise/templates/exercise/staff/results.html

```diff
@@ -15,17 +15,18 @@
 {{ block.super }}
 <!--Library to handle multiselection on select tags-->
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/css/bootstrap-multiselect.css">
-<script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/js/bootstrap-multiselect.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-multiselect/0.9.15/js/bootstrap-multiselect.js"></script>
 
-<!--Required libraries for exporting table data to xlsx, csv or txt.-->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/css/tableexport.min.css">
-<script src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.14.3/xlsx.core.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/TableExport/5.2.0/js/tableexport.min.js"></script>
+<!-- DataTables plugin with Bootstrap 3 styling and Buttons/export support -->
+<!-- To upgrade plugin versions, go to https://datatables.net/download/index -->
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/jszip-2.5.0/dt-1.10.24/b-1.7.0/b-html5-1.7.0/datatables.min.css"/>
```

Comment:
https://datatables.net/download/compatibility says: "DataTables and its extensions all require jQuery 1.7 or newer. Like all software, it is recommended that you always use the latest version of jQuery that is available as it will include additional features, performance enhancements and bug fixes."

In my 10+ year journey with DataTables it has made very little difference which jQuery version I've used. Typically the problems have occurred when using copy-pasted code examples that may result in mixing the old and new DataTables APIs on the same page.

It may be safest, though, to host it locally instead of loading from the CDN.